### DISCUSSION TITLE
Contributors Role Requests

### DISCUSSION THREAD

**Discussion by Sceleratis** [Epix-Incorporated/Adonis#433] [General]
If you contributed and need the rank in the group/role on the discord, post your Roblox & Discord usernames here so I know it's actually you and I'll do the thing.

> [!IMPORTANT]
> To help speed up the process, please also include a link to the pull request of your contribution (e.g. #123) for ease of verification.

**Comment by ccuser44** [comment]


**Reply by ccuser44** [reply]
If it were possible could my alt also be added. My alt is : Ales_Boi

**Reply by Sceleratis** [reply]
Main was done. Alt doesn't appear to be in the group.

**Reply by ccuser44** [reply]
Ok Ill add the alt to group

thx

**Reply by ccuser44** [reply]
Ok added the alt to the group

**Reply by Sceleratis** [reply]
Ranked.

**Reply by ccuser44** [reply]
Thanks scel

**Comment by P3tray** [comment]
P3tray
P3tray#4828

Both are already done, simply adding for reference.

**Comment by Expertcoderz** [comment]
_(Already ranked)_

**Discord:** Expertcoderz#9517
**Roblox:** Expertcoder2 (display name: Expertcoderz)

**Comment by policetonyR** [comment]
roblox: t1dtony
discord: tony#9999 (already roled)

**Reply by Sceleratis** [reply]
Ranked

**Comment by Webm07** [comment]
Roblox: Awesomewebm
Discord: N/A

**Reply by Sceleratis** [reply]
Ranked

**Comment by ghost** [comment]
My discord is Subby#7842 and my username is softfoxpaws I wish to be ranked

**Reply by Sceleratis** [reply]
Ranked

**Comment by 8stakka** [comment]


**Reply by Sceleratis** [reply]
Gave you the Discord role, can't rank since not in the group

**Reply by 8stakka** [reply]
> Gave you the Discord role, can't rank since not in the group

Joined the group.

**Reply by Sceleratis** [reply]
> 
> 
> > Gave you the Discord role, can't rank since not in the group
> 
> Joined the group.

Ranked.

**Comment by alau740** [comment]
roblox: windoesboy9 (display name: Windoes)
discord: ali8888#8508

**Comment by DJKieranl29** [comment]
kieranl29_offical#3816

**Comment by Webm07** [comment]
Discord: Awesomewebm#5430

**Comment by Bizarre-source** [comment]
Roblox : Bizarre / Malcar_Llyon
Discord : Bizarre#1756

**Comment by oceansync** [comment]
Roblox : Deniernal354
Discord : Deni#0274

**Comment by jun022222222** [comment]
jun022222222
jun022222222#3638

**Comment by h0nzzz** [comment]
roblox: lab_minecraft11
discord: honzzz#3018 (role given already)

**Comment by Smaltin** [comment]
Roblox: @Smaltin (Shirakami_Fvbuki)
Discord: Smaltin#2208

Also need to be added to Credits.lua, contributed PR 664

**Comment by 2n3d** [comment]
Discord: fx#0009
Roblox: AviatorFX

**Comment by xvarmkorv2** [comment]
Discord: xVarmkorv2#9278
Roblox: xvarmkorv2

**Comment by Redvoodoo** [comment]
Discord : Dobby#3569
Roblox : redmage9

**Reply by Sceleratis** [reply]
Remind me what you contributed as it's not listing you as a repo contributor and I don't remember.

**Comment by Khaomi** [comment]
```lua
self.Name = "hayper1919"
self.Discord = "hayper#6879"
```

Roblox Username: hayper1919
Discord: hayper#6879

**Comment by GGshor** [comment]
Roblox: GGshor
Discord: GGshor#8863

**Comment by WideManHost** [comment]
roblox: epicrunner3451
discord: Wide Wade#7330

**Comment by autodoorsdev** [comment]
Roblox: automaticdoors315
Discord: Door#3773

**Comment by nikothegamer100** [comment]
Roblox: nikothegamer100
Discord: nikothegamer#5275

**Comment by Lethalitics** [comment]
Roblox: Lethalitics
Discord: Lain#6777

**Comment by RoyallyFlushed** [comment]
Roblox: RoyallyFlushed
Discord: Royal#1234

**Comment by supercoolspy** [comment]
Roblox: supercool_spy
Discord: SuperCoolSpy#0002

**Comment by ghost** [comment]
Already ranked in roblox, just need to be rolled in discord (new account)
Pintora#4213

**Comment by Daniel19341** [comment]
Roblox:  Alexissomuchcooler20
Discord: JacksontheHero#0195

**Comment by Gamerlife5704** [comment]
Discord: gamerlife5704#3407
Roblox: gamerlife5704

**Comment by watameln** [comment]
lbe07
watameln#7484

**Comment by NoobBucket** [comment]
Roblox: @techally
Discord: KFC_Bucket#1423

**Comment by Z3phrz** [comment]
Roblox: AlexanderDarmody
Discord: Alex#5200

**Comment by DaRealGandhi20** [comment]
Roblox: Gandhi20
Discord: Gandhi20#8928

**Comment by WalkerOfBacon** [comment]
Roblox: Jumbo891011
Discord: baconfunds

**Comment by Daniel19341** [comment]
Roblox: Daniel_19341
Discord: simplly_daniel

**Reply by Expertcoderz** [reply]
Could we confirm which contribution you made? You don't seem to have opened any PRs in this repository before.

**Comment by SuperCater** [comment]
Roblox: CaterSuper
Discord: cater

**Comment by dylanjkl** [comment]
Roblox: dylanjkl
Discord: dylanjkl

**Comment by kaiserandaxl** [comment]
Roblox: kaiserandaxl

**Comment by sqikerz** [comment]
Roblox: Sqikerz
Discord: spikers

**Comment by convxors** [comment]


**Reply by convxors** [reply]
@watameln I've contributed to PR #1523 by fixing a typo. It was approved by a maintainer.

**Comment by wilsontulus** [comment]
Roblox: wilsontulus5 (joined the group)
Discord: wilsontulus (verified in server)

**Comment by convxors** [comment]
Roblox: Scripted_Connor
Discord: scripted_connor

#1593 
#1597

**Comment by cl1ents** [comment]
Roblox: ev1
Discord: evpn

#1603 (i hope!)
#1516
#1410 (technically, someone just 1upped me in opening the PR, but I did write the code in the issue mentioned in the pr lol)

**Comment by supercoolspy** [comment]
Roblox: supercool_spy
Discord: @spyied (already have contributor)

**Comment by xs4u** [comment]
_(given)_

Roblox: RjavisDroutDeveloper
Discord: xs4u

**Comment by annoynumus9** [comment]
_(completed)_
Roblox user: @churo98789
Discord user: "anonymous1253." *including the period in the quotes

**Comment by duckymomo360** [comment]
_(completed)_
Roblox: quinowenryanjen
Discord: duckymomo360

**Comment by GuestDaProtogen** [comment]
_(completed)_
Roblox: GuestDaProtogen
Discord: guestdaprotogen

#2000

**Comment by AidenJamesYt** [comment]
(completed)
Roblox: AidenJamesYt
Discord: aidenjamesyt
Contribution: #2116 #2133

**Comment by thatgato** [comment]
completed

**Comment by realdint** [comment]
roblox dintisepic
discofrd _dint
https://github.com/Epix-Incorporated/Adonis/pull/2140

**Comment by Kanuin** [comment]
Roblox Kanuin111
Discord bali117
#2182

**Comment by root0055** [comment]
Roblox : root0055
Discord : rootrblx
#2155 

Also if possible to be added  to Credits.luau.
